src/operator/tensor/broadcast_reduce_op_index.cc

```diff
@@ -87,6 +88,40 @@ Example::  
 .add_argument("data", "NDArray-or-Symbol", "The input array");
 
 NNVM_REGISTER_OP(pick)
+.describe(R"code(Picks elements from an input array according to the input indices along the given axis.
+
+Given an input array of shape ``(d0, d1)`` and indices of shape ``(i0,)``, the result will be
+an output array of shape ``(i0,)`` with::
+
+  output[i] = input[i, indices[i]]
+
+By default, if any index mentioned is too large, it is replaced by the index that addresses
+the last element along an axis.
+
+Pick operator supports n-dimensional input and (n-1)-dimensional indices arrays.
```

Comment:
What does "Pick operator" mean? Is pick the name of the operator? Shoudl it then be a function name?